screens/popups/transfer_pokemon/transfer.gui_script

```diff
@@ -326,13 +326,13 @@ function on_input(self, action_id, action)
 		if self.move_button_enabled then
 			a = gooey.button("btn_move", action_id, action, function() move(self) end, function(b) gooey_buttons.common_button(b, gui.get_node("txt_move")) end)
 			c = gooey.button("btn_release", action_id, action, function() release(self) end)
-			d = gooey.button("btn_share", action_id, action, function() share.export(self.pokemon_id) end, gooey_buttons.share)
 		end
+		d = gooey.button("btn_share", action_id, action, function() share.export(self.pokemon_id) end, gooey_buttons.share)
 		gooey.button("btn_close", action_id, action, back, gooey_buttons.close_button)
 		gooey.button("btn_cancel", action_id, action, function() cancel_swap(self) end, gooey_buttons.close_button)
 		local g = gooey.button("background", action_id, action, function() end)
 
-		if not a.over and not b and not c.over and not d.over and not g.over then
+		if (not self.move_button_enabled or not a.over) and not b and (not self.move_button_enabled or not c.over) and not d.over and not g.over then
```

Comment:
Do things like `a` and `c` have `over` set even when they are not enabled?

Reply:
In this case if the button is not enabled the if statement on line `326` will not be entered and the `a` and `c` (which should be refactored in be actual names of the button) will be `nil`. I debated with doing `(a == nil or not a.over)` or like this `(not self.move_button_enabled or not a.over)` but thought this would be more what I actually wanted to tell with the code.